Automattic/wp-calypso
# Store: Add Low Inventory widget to the dashboard

See #16732 – this PR adds a new widget to the dashboard, on both the no-orders & has-orders view, to show products with low or no inventory.

Low stock, on "has orders" dashboard, sits in next to the reports widget. This location might need rethinking if the stats widget is meant to replace the reports widget (#23344)
<img width="910" alt="screen shot 2018-03-15 at 4 52 55 pm" src="https://user-images.githubusercontent.com/541093/37490916-8017a750-2872-11e8-8a18-459cbc2846e0.png">

No stock on a product turns the border red, and this is on the "no orders" view, where it replaces the stats widget (unless there are no products).
<img width="906" alt="screen shot 2018-03-15 at 5 05 55 pm" src="https://user-images.githubusercontent.com/541093/37491128-292e0dd4-2873-11e8-936d-aa21209fa4c4.png">

This is the view when there are no products low or out of stock, maybe we could have an illustration here?
<img width="451" alt="screen shot 2018-03-15 at 4 41 10 pm" src="https://user-images.githubusercontent.com/541093/37490918-803f4ef4-2872-11e8-9c00-6f46b6631ff5.png">

This doesn't add the initial view (which I think instead should be the "inventory management is disabled" view), or the configuration options. I'll follow up with another PR for that.

**To test**

- View your dashboard, `/store/:site`
- If you have products, you should see this new widget
- Low or No stock is determined by the settings in `/wp-admin/admin.php?page=wc-settings&tab=products&section=inventory`
- You can change product stock quantity to see the different states: low stock, no stock, everything's OK
